packages/field-base/src/input-slot-mixin.js

```diff
@@ -52,8 +52,8 @@ const InputSlotMixinImplementation = (superclass) =>
     }
 
     /** @protected */
-    connectedCallback() {
-      super.connectedCallback();
+    ready() {
+      super.ready();
```

Comment:
Changed to have the reference to the input stored before running "delegation" and "constraints" observers.